js/ui/button.js

```diff
@@ -314,6 +330,29 @@ var Button = Widget.inherit({
         };
     },
 
+    _isToggleable: function() {
+        return this.option("toggleable");
+    },
+
+    _updateToggle: function() {
+        if(this._isToggleable()) {
+            var toggle = this.option("selected");
+            this.option("selected", !toggle);
+        }
+    },
+
+    _renderToggle: function() {
+        if(this._isToggleable()) {
+            this.$element().toggleClass(BUTTON_STATE_SELECTED_CLASS, !!this.option("selected"));
+        }
+    },
+
+    _attachFeedbackEvents: function() {
```

Comment:
Why are we skip this method? What if a widget was created with toggleable === false and later this option will be changed for "true"?

Reply:
The active state should not apply in the toggleable mode by Dmitry Kharkov design. Your scenario works correctly and I have tests on that.